.circleci/config.yml

```diff
@@ -24,10 +24,13 @@ jobs:
           key: v1-node-tslib-deps-{{ checksum "tslib/yarn.lock" }}
       - run: cd admin; yarn install --frozen-lockfile
       - run:
-          name: Install JUnit coverage reporter
+          name: Admin lint and ts compile
+          command: cd admin; yarn lint; rm -rf dist
+      - run:
+          name: Install JUnit coverage reporter for admin unit tests
           command: cd admin; yarn add --dev jest-junit
       - run:
-          name: Run tests with JUnit as reporter
+          name: Admin unit tests with JUnit as reporter
           command: cd admin; ./node_modules/.bin/jest --ci --runInBand --reporters=default --reporters=jest-junit --collectCoverage=true
```

Comment:
As far as I know, CircleCI is not surfacing any code coverage details - so why bother to collect code coverage?

Reply:
We get this...
![Screenshot 2020-06-09 10 51 10](https://user-images.githubusercontent.com/329532/84133487-2645ed80-aa3f-11ea-95d6-c0f87d0c4697.png)

Ideally i'd like a gate on the overall coverage metric, which i think would be suitable for codacy?

Reply:
Fair enough then